libs/blocks/chart/list.js

```diff
@@ -39,12 +41,9 @@ export function listChartData(json) {
   return data;
 }
 
-export const getListHtml = (data) => {
-  // ToDo: Replace firstList with carousel iterator MWPW-114550
-  const firstList = data[0];
-  const listType = firstList.type?.toLowerCase() === 'numbered' ? 'ol' : 'ul';
-
-  const listItems = firstList.list.reduce((prev, { name = '', extra, image, alt = '' }) => (
+export const getListHtml = (list) => {
+  const listType = list.type?.toLowerCase() === 'numbered' ? 'ol' : 'ul';
+  const listItems = list.list.reduce((prev, { name = '', extra, image, alt = '' }) => (
```

Comment:
`list.list` is confusing. Perhaps the param should be `chart` or `data`

Reply:
`chart` sounds good, `data` could be confusing with the other `data`.